projects/storefrontlib/src/lib/checkout/store/reducers/checkout.reducer.ts

```diff
@@ -10,8 +11,18 @@ export interface CheckoutState {
   orderDetails: any;
 }
 
+export const emptyAddress: Address = {
+  titleCode: '',
+  firstName: '',
+  lastName: '',
+  line1: '',
+  town: '',
+  postalCode: '',
+  country: { isocode: '' }
+};
+
 export const initialState: CheckoutState = {
-  address: {},
+  address: emptyAddress,
```

Comment:
Can we just use `{}`?

Reply:
Yes

Reply:
It was only like that because the previous `Address` type used did not have all fields marked as optional.